CONTRIBUTING.md

```diff
@@ -0,0 +1,269 @@
+
+### Documentation Contributions
+
+ownCloud is very proud of the documentation it has, which is the work of a great team of people. Of course, also the documentation is open to contributions.
+
+See the [Getting Started Guide](https://github.com/owncloud/docs/blob/master/docs/getting-started.md) on how to get started. Many other useful information is summarized in the [Documentation Readme](https://github.com/owncloud/docs).
+
+### Internationalization
+
+ownCloud is translated in many languages to allow people from all oever the world to use ownCloud in their native language. For translations, ownCloud uses [Transifex](https://www.transifex.com) as community based collaboration collaboration platform for internationalization.
+
+For contributions please refer to the [Transifex Resources](https://www.transifex.com/resources/) to learn how to improve ownClouds translations there.
+
+### Deployments: Docker, Ansible and friends
+
+Depending on the ownCloud component, there is complex deployment tooling to install in various environments. There is for example [ownCloud Ansible](https://github.com/owncloud-ansible) with Ansible resources. Contributions to that are very appreciated and follow the same guidelines as every other code contribution.
+
+## Styleguides
+
+To keep up with a consistent code and tooling landscape, some of the ownCloud modules maintain styleguides for contributions. It is mandatory to follow them in contributions.
+
+### Git Commit Messages
+
+* Use the present tense ("Add feature" not "Added feature")
+* Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
+* Limit the first line to 72 characters or less
+* Reference issues and pull requests liberally after the first line
+* When only changing documentation, include `[docs-only]` in the commit title
+
+### Golang Styleguide
+
+TBD
+
+### Web Styleguide
+
+See the styleguide document in the [ownCloud Web Repository](https://github.com/owncloud/web).
+
+#### Documentation Styleguide
+
+Find the [ownCloud Docuemntation Styleguide](https://github.com/owncloud/docs/blob/master/docs/style-guide.md).
+
+## Additional Notes
+
+### Issue and Pull Request Labels
+
+This section lists the labels we use to help us track and manage issues and pull requests. Most labels are used across all ownCloud repositories, but some are specific.
+
+
+[GitHub search](https://help.github.com/articles/searching-issues/) makes it easy to use labels for finding groups of issues or pull requests you're interested in. To help you find issues and pull requests, each label can be used in search links for finding open items with that label in the ownCloud repositories.
+
+We  encourage you to read about [other search filters](https://help.github.com/articles/searching-issues/) which will help you write more focused queries.
```

Comment:
```suggestion
We encourage you to read about [other search filters](https://help.github.com/articles/searching-issues/) which will help you write more focused queries.
```
aaand this is the same link as above? redundant? drop? or replace with an example that searches for a specific label.

Reply:
removed that paragraph. Redundant indeed.